### PR TITLE
Analysis #18- vacant lots analysis- arlington

### PR DESCRIPTION
**Pull request by sstrochak**

### CODE REVIEW

**Comment by awunderground** [comment]
1) `"../Macros/read-jurisdiction.R")` does not exist in this branch right now. So the code does not run. 
2) Keep the same amount of "what" in the text, but please add more "why". I'm not always sure why you are doing each step. 
    * For example, why is CRS 4326 used?
    * It's probably worth using more "###" headers so it is easier to navigate. 
3) The `.Rmd` does not include any programmatic checks. Maybe program the CRS check?
4) It's peculiar how the file ends by assigning a tibble called `test_all`. Shouldn't we see the contents of this tibble in the `.Rmd`?
5) @lhendey has requested that we overwrite data structures less since we don't have logs with row counts or summary statistics.

**Comment by awunderground** [comment]
1) Always include argument names when calling custom functions like `get_unit_summary()` and `get_unit_variables()`.

For example:

```
get_unit_summary(dataset = zoningdata1,
                 construction_efficiency = 0.7,
                 average_unit_size = 940,
                 dwelling_type_breakdown = TRUE)
```

2) It would be great to have some checks after the spacial join. I don't know how to evaluate if that part of the code is behaving as anticipated. 
3) Why not write the following as `case_when()`?

```
vacantfinal <- vacantfinal %>% 
  mutate(zoning_code = ifelse(zoning == "RC",
                              "R-C",
                              zoning),
         zoning_code = ifelse(zoning_code %in% c("C-O-ROSSLYN", "C-O-ROSS"),
                              "C-O Rosslyn",
                              zoning_code),
         zoning_code = ifelse(zoning_code == "RA-H-3.2",
                              "RA-H 3.2",
                              zoning_code))
```

**Comment by sstrochak** [comment]
@awunderground what is most helpful in testing the spatial join? The percent of parcels that joined?

**Comment by sstrochak** [comment]
@lhendey probably ready for you to take a look- let me know if you there is another metric you'd like to see on the spatial join (I added a few checks). Only look at vacant-lots-arlington.Rmd

File: Prog/vacant-lots-arlington.Rmd [hunk 1]
```diff
@@ -180,40 +161,436 @@ First, we look at all the options, and filter out all the observations marked as
...
+  mutate(zoning_true = ifelse(count == 1,
+                              map(zlist, max, na.rm = TRUE),
+                              ""))
+```
+
+Priotize codes that are purely residential (begin with R). If there are more than 1 "R" zoning codes, we want to take the one with the lower number (take the lower density code, so we do not overestimate, since part of the building will be in the lower density area). We do this by extracting the numbers from the zoning codes, taking the minimum, and then selecting the element that contains that number.
+
+```{r true2}
+
+vacantweird2 <- vacantweird2 %>%
```
Comment (sstrochak) [inline Prog/vacant-lots-arlington.Rmd]: @awunderground would like feedback on this step- it gets the job done, but I feel like there could be a better solution.

**Comment by sstrochak** [comment]
@pspauster @cdavisurban this is merged

**Pull request merged**